template/cordova/lib/PluginHandler.js

```diff
@@ -85,12 +85,12 @@ var handlers = {
             var src = obj.src;
             var dest = src;
             var type = obj.type;
+            var targetDir = obj.targetDir || '';
```

Comment:
Consider keeping this variable inside the else statement since it applies there only.